LunrCoreLmdb/SerializationExtensions.cs

```diff
@@ -0,0 +1,254 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Lunr;
+
+namespace LunrCoreLmdb
+{
+    public static class SerializationExtensions
+    {
+        #region Vector
+
+        public static ReadOnlySpan<byte> Serialize(this Vector vector)
+        {
+            var ms = new MemoryStream();
+            var bw =new BinaryWriter(ms);
+            var context = new SerializeContext(bw);
```

Comment:
Should any of those be `using`s?

Reply:
Most likely, yes. I made the change. I think when I wrote it I was thinking that because I'm using `GetBuffer` to try to avoid an allocation to copy the bytes out of `MemoryStream`, that disposal scoping would cause the buffer to dispose prematurely, but in practice that's not a real thing.

Reply:
Fixed.